src/app/catalogue/Catalogue.java

```diff
@@ -28,14 +31,14 @@ public Catalogue() {
      * @param item This is the new Item object to be added to the Catalogue map list.
      */
     public void addItemToList(Item item) {
-        List<Item> list = this.catalogueMap.get(item.getType());
-
-        if (list == null) {
-            list = new ArrayList<>();
-            this.catalogueMap.put(item.getType(), list);
+        if (item != null) {
```

Comment:
If item is null then escape the method. Escaping early allows for easier to read code and prevents you from needing to add another layer of indentation